packages/@aws-cdk/aws-cloudtrail/test/integ.cloudtrail-supplied-bucket.lit.ts

```diff
@@ -31,6 +33,7 @@ Trailbucket.addToResourcePolicy(new iam.PolicyStatement({
 
 const trail = new cloudtrail.Trail(stack, 'Trail', {bucket: Trailbucket});
 
+trail.addLambdaEventSelector([lambdaFunction.functionArn]);
```

Comment:
Please run `yarn test` after making this change. It will tell you the command you need to run to update the integ test golden master.

Reply:
@rix0rrr After making a few adjustments, both `yarn test` and `yarn integ` are passing when run locally.

```bash
% yarn test
yarn run v1.22.4
$ cdk-test

test.cloudtrail.js
✔ constructs the expected resources - with no properties
✔ constructs the expected resources - with s3bucket
✔ constructs the expected resources - with imported s3 bucket
✔ constructs the expected resources - with s3KeyPrefix
✔ constructs the expected resources - with cloud watch logs - enabled
✔ constructs the expected resources - with cloud watch logs - enabled and custom retention
✔ constructs the expected resources - with event selectors - with default props
✔ constructs the expected resources - with event selectors - with hand-specified props
✔ constructs the expected resources - with event selectors - with management event
✔ constructs the expected resources - with event selectors - for Lambda function data event
✔ constructs the expected resources - with event selectors - for all Lambda function data events
✔ add an event rule

OK: 39 assertions (1984ms)
```

```bash
% npm run integ

> @aws-cdk/aws-cloudtrail@0.0.0 integ /Users/rbright/Projects/aws-cdk/packages/@aws-cdk/aws-cloudtrail
> cdk-integ

Trying to deploy integ.cloudtrail-supplied-bucket.lit.js
Selected stack: integ-cloudtrail
Success! Writing out reference synth.
Cleaning up.
Trying to deploy integ.cloudtrail.lit.js
Selected stack: integ-cloudtrail
Success! Writing out reference synth.
Cleaning up.
```

However, the CodeBuild run fails with:

```
@aws-cdk/aws-cloudtrail: Error: Some stacks have changed. To verify that they still deploy successfully, run: 'npm run integ integ.cloudtrail-supplied-bucket.lit.js integ.cloudtrail.lit.js'
```

Could you provide more insight on what needs to be done to remedy this?

Reply:
Nevermind, `*.expected.json` files were generated as part of the local runs and needed to be committed. I suspect that will resolve this issue.